flink-dist/src/main/assemblies/bin.xml

```diff
@@ -108,6 +108,13 @@ under the License.
 		</file>
 
 		<!-- Built-in connectors and formats -->
+		<file>
+			<source>../flink-connectors/flink-connector-base/target/flink-connector-base-${project.version}.jar</source>
```

Comment:
This is unnecessary; it could just be included in the flink-dist jar.

Reply:
Moved to flink-dist.